ledger/src/blockstore_db.rs

```diff
@@ -1444,9 +1462,31 @@ fn get_cf_options<C: 'static + Column + ColumnName>(
         });
     }
 
+    process_cf_options_advanced(&mut cf_options, &options.column_options);
+
     cf_options
 }
 
+fn process_cf_options_advanced(cf_options: &mut Options, column_options: &LedgerColumnOptions) {
+    if column_options.dynamic_size_with_fixed_level {
+        // Limiting the number of levels to three allows us to answer read requests
+        // with capped read amplification.  From the data-points in mainnet-beta,
+        // the majority of the data can fit in three levels.  For those data which
+        // does not fit within three levels, we use the next option to dynamically
+        // change the size of each level to allow the level structure to be adapted
+        // over time.
+        cf_options.set_num_levels(DYNAMIC_SIZE_MAX_NUM_LEVELS);
+
+        // This option allows rocksdb to dynamically change the size of each level
+        // to reach the target number of levels.  This setting together with the
+        // above `set_num_levels` allows the column family to be packed within
+        // three levels -- no matter whether the column familly is big or small.
+        // Note that this option cannot be used to reopen an existing db without
+        // this setting.
```

Comment:
"Note that this option cannot be used to reopen an existing db without this setting"

Hmm does this mean we can't turn on this option when opening any existing ledger right now?

Reply:
Based on the RocksDB document, it's okay but the LSM structure will not be shaped as expected.

Reply:
Got it, could we clarify this comment then? Right now it seems to imply if you didn't open RocksDb with this option initially, you should never open it with this option later